src/EditorFeatures/CSharpTest/Completion/CompletionProviders/ObjectCreationCompletionProviderTests.cs

```diff
@@ -437,5 +438,147 @@ void M1()
 ";
             await VerifyItemExistsAsync(markup, "object");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task CommitOnAngleBracket_Simple()
+        {
+            var before = @"
+namespace NS
+{
+    public class A { }
+}
+
+public class C
+{
+    private NS.A _a = new $$
+}
+";
+            var after = @"
+namespace NS
+{
+    public class A { }
+}
+
+public class C
+{
+    private NS.A _a = new NS.A<
+}
+";
+            await VerifyProviderCommitAsync(before, "NS.A", after, '<', string.Empty);
```

Comment:
I would not do any commit tests in the Provider test framework.  Commit tests here are very busted as they don't actually invoke our real commit infrastructure.  Commit tests shuold be in the CompletionCommandHandlerTests.  @rchande can you help port these?